### DISCUSSION TITLE
Information Sofar2MQTT supports Voltronic King

### DISCUSSION THREAD

**Discussion by saviothecnic** [softwarecrash/Solar2MQTT-ESP8266#55] [General]
Your work among the protocols is very interesting. I don't seem to have seen the support for Voltronic KING inverters
You can also know what you have as information in MQTT
and what can be sent via MQTT?

**Comment by softwarecrash** [comment]
without the communication protocol i cant say anything

**Reply by saviothecnic** [reply]
the protocol should be PIP30
Except that the KING series has different menus than the other Voltronics

**Reply by softwarecrash** [reply]
When it ist pi30 protocol it will Work with my Software.

The menu and other on the device iself have nothing to so with the communication

**Reply by saviothecnic** [reply]
Ok perfect you have a list of the data I can receive in MQTT
and also those that I can send in MQTT
I am very interested if it is possible to change parameter 10
which is the operating logic and the parameter that changes the charging priority from solar only to Utilities and solar
Read also parallel inverters

**Reply by softwarecrash** [reply]
For device Control, pleased read the wiki
